packages/cli/src/commands/cluster/add/setupLinkerd.ts

```diff
@@ -75,15 +76,28 @@ export async function setupLinkerd(
         );
       },
     },
-    // TODO: @seth - Ensure the kubectx is correct for Linkerd
+    {
+      title: "Set Kubernetes Context",
+      task: async () => {
+        if (!kubeConfigContext) {
+          throw new CLIError("Kube config context is not set");
+        }
+        await execute({
```

Comment:
This seems wrong as it will break if multiple cluster installs are running concurrently.

Reply:
🤦🏻‍♂️ I looked at the flags available on the command and didn't see a way to pass it in (felt wrong). Looking again it's a global flag which isn't referenced on individual commands. Will fix!